docs/tempo/website/configuration/_index.md

```diff
@@ -738,3 +739,13 @@ overrides:
     # Ingestion strategy, default is `local`.
     ingestion_rate_strategy: <global|local>
 ```
+
+## Search
```

Comment:
Should we mention current limitations. i.e. ingester search only?

Reply:
Yes was thinking we need to mention that, but didn't think in the config section.   How about a new "Search" section?

Reply:
+1 for a section/page dedicated to search

Reply:
After some consideration, went with a new page in the Getting Started section that goes through the high-level Tempo/Grafana features and their availability.  It is somewhat modeled after the Loki page: https://grafana.com/docs/loki/latest/getting-started/grafana/

Thoughts?